packages/openneuro-server/src/graphql/schema.js

```diff
@@ -161,6 +161,13 @@ export const typeDefs = `
     reexportRemotes(datasetId: ID!): Boolean
     # Reset draft commit
     resetDraft(datasetId: ID!, ref: String!): Boolean
+    # Bulk delete individual files
+    deleteBulk(datasetId: ID!, files: [DeleteFile]): Boolean
```

Comment:
Can we consolidate the delete APIs down to one deleteFiles API? The other cases map pretty well to this implementation.